jobs/build/refresh-images/Jenkinsfile

```diff
@@ -117,7 +117,11 @@ node('openshift-build-1') {
   """
 
               buildlib.oit """
---working-dir ${OIT_WORKING} --group openshift-${OSE_MAJOR}.${OSE_MINOR} --include aos3-installation-docker \\
+--working-dir ${OIT_WORKING} --group openshift-${OSE_MAJOR}.${OSE_MINOR} \\
+--include aos3-installation-docker \\
```

Comment:
optional-includes

Reply:
You mean the ones right below on the next line?